resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/KubernetesClusterSchedulerBackend.scala

```diff
@@ -107,6 +118,11 @@ private[spark] class KubernetesClusterSchedulerBackend(
           .withLabel(SPARK_APP_ID_LABEL, applicationId())
           .withLabel(SPARK_ROLE_LABEL, SPARK_POD_EXECUTOR_ROLE)
           .delete()
+       kubernetesClient
```

Comment:
indentation?

Reply:
It seems that we can have a `configMap` leftover if the pod deletion fails. Could you do this configMap deletion in a new `Utils.tryLogNonFatalError` block?